src/components/SubCategorySummary/SubCategorySummary.scss

```diff
@@ -1,5 +1,9 @@
 @import '~style/variables';
 
+.wrapper {
+  border-top: 4px solid $color-white;
```

Comment:
No strictly related, but added the little white gap that was added to the designs at some point.